From 4e1dac79637c3e4f1e606a3a7fcb6dd2974de7c5 Mon Sep 17 00:00:00 2001
From: Christian Costa <titan.costa@gmail.com>
Date: Thu, 8 Jan 2015 00:16:39 +0100
Subject: wined3d: Use proper color key type define when calling
 wined3d_texture_set_color_key.

Fix bug https://bugs.winehq.org/show_bug.cgi?id=37748 which was a regression caused
by commit a8ab56941577a01e17b947cddd374ba52ffd8245.
---
 dlls/wined3d/surface.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/dlls/wined3d/surface.c b/dlls/wined3d/surface.c
index fb27c8a..ad41834 100644
--- a/dlls/wined3d/surface.c
+++ b/dlls/wined3d/surface.c
@@ -3687,12 +3687,12 @@ static HRESULT surface_blt_special(struct wined3d_surface *dst_surface, const RE
         else if (flags & WINEDDBLT_KEYSRCOVERRIDE)
         {
             /* Use color key from DDBltFx */
-            wined3d_texture_set_color_key(src_surface->container, WINEDDSD_CKSRCBLT, &DDBltFx->ddckSrcColorkey);
+            wined3d_texture_set_color_key(src_surface->container, WINEDDCKEY_SRCBLT, &DDBltFx->ddckSrcColorkey);
         }
         else
         {
             /* Do not use color key */
-            wined3d_texture_set_color_key(src_surface->container, WINEDDSD_CKSRCBLT, NULL);
+            wined3d_texture_set_color_key(src_surface->container, WINEDDCKEY_SRCBLT, NULL);
         }
 
         surface_blt_to_drawable(device, filter,
@@ -3700,7 +3700,7 @@ static HRESULT surface_blt_special(struct wined3d_surface *dst_surface, const RE
                 src_surface, src_rect, dst_surface, dst_rect);
 
         /* Restore the color key parameters */
-        wined3d_texture_set_color_key(src_surface->container, WINEDDSD_CKSRCBLT,
+        wined3d_texture_set_color_key(src_surface->container, WINEDDCKEY_SRCBLT,
                 (old_color_key_flags & WINEDDSD_CKSRCBLT) ? &old_blt_key : NULL);
 
         surface_validate_location(dst_surface, dst_surface->container->resource.draw_binding);
-- 
2.2.1

